dashboard/app/views/admin_reports/funometer.html.haml

```diff
@@ -1,100 +1,77 @@
 - content_for :head do
   %script{src:'https://www.google.com/jsapi'}
 
-%h1
-  Fun-O-Meter Stats
+- if @stats.present?
+  %h1
+    Fun-O-Meter Stats
 
-%h2
-  Overall Percentage:
-  %span= number_to_percentage(@overall_percentage, precision: 2)
+  %h2
+    Overall Percentage:
+    %span= number_to_percentage(@stats['overall'], precision: 2)
 
-%br
+  %br
 
-= render 'funometer_percentages_by_day', :percentages_by_day => percentages_by_day
+  -# Renable the graphs after reconciling the different object types.
```

Comment:
The _funometer_percentages_by_day.html.haml was created to reuse code between the top-level view, the script view, and the level view. All three (before this changed) passed the result of the SQL queries, rather than hashes or arrays. In the cron, we need to be saving the hash or array. So (a) the other dashboards need to be changed to use hashes or arrays or (b) the code can't be reused.

Right?